glances/glances.init

```diff
@@ -43,7 +43,7 @@ case "$1" in
         else
             sudo -u "$user" $cmd >> "$stdout_log" 2>> "$stderr_log" &
             sleep 1 # process not always started fast enough
-            echo $(pgrep -U "$user" -xf "$cmd") > "$pid_file"
```

Comment:
These might actually need the quotes that I removed. ex: `echo "$(pgrep -U "$user" -xf "$cmd")" > "$pid_file"`